packages/docs/stories/components/range.stories.ts

```diff
@@ -0,0 +1,453 @@
+  `,
+};
+
+export const Sizes: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('range', 'size'),
+      },
+    },
+  },
+  render: () => html`
+    <div class="size-wrapper">
+      <syn-range label="Small" max="100" min="0" size="small" value="33"></syn-range>
+      <syn-range label="Medium" max="100" min="0" size="medium" value="66"></syn-range>
+      <syn-range label="Large" max="100" min="0" size="large" value="99"></syn-range>
+    </div>
+    <style>
+      .size-wrapper {
+        display: flex;
+        flex-direction: column;
+        gap: var(--syn-spacing-medium);
+      }
+    </style>
+  `,
+};
+
+export const PrefixSuffixText: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('range', 'prefix-suffix'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-range
+      help-text="Controls the volume of the current song"
+      label="Volume"
+      max="100"
+      min="0"
+      value="50"
+    >
+      <span slot="prefix">0</span>
+      <span slot="suffix">100</span>
+    </syn-range>
+
+    <p style="margin: var(--syn-spacing-x-large) 0;">This can be used to add input fields or icons.</p>
+
+    <syn-range label="Estimated Time" class="suffix-input-field" min="0" max="60" value="30">
+      <span slot="prefix">0</span>
+      <span slot="suffix">
+        <div class="suffix-input">
+          60
+          <syn-input value="30" type="number" no-spin-buttons min="0" max="60">
+            <span slot="suffix">sec</span>
+          </syn-input>
+        </div>
+      </span>
+    </syn-range>
+    
+    <style>
+      .suffix-input {
+        align-items: center;
+        display: flex;
+        gap: var(--syn-spacing-medium);
+        syn-input {
+          min-width: 0;
+        }
+      }
+    </style>
+    <script>
+      [...document.querySelectorAll('.suffix-input-field')]
+        .forEach(range => {
+          const input = range.querySelector('syn-input');
+
+          range.addEventListener('syn-input', e => {
+            const { target } = e;
+            if (target.tagName !== 'SYN-RANGE') return;
+            target.querySelector('syn-input').value = target.value;
+          });
+
+          input.addEventListener('syn-input', e => {
+            const { target } = e;
+            target.closest('syn-range').value = target.value;
+          });
+        });
+    </script>
+  `,
+};
+
+export const CustomTrackColors: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('range', 'custom-track-colors'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-range
+      class="custom-track-color"
+      max="100"
+      min="0"
+      value="50"
+    ></syn-range>
+    <style>
+      .custom-track-color {
+        --track-color-active: var(--syn-color-success-600);
+      }
+    </style>
+  `,
+};
+
+export const CustomTrackOffset: Story = {
```

Comment:
![CleanShot 2024-07-16 at 10 30 10](https://github.com/user-attachments/assets/ac7ded60-128d-4b50-a602-4d26cc2a06b1)

Maybe not correct in design, but it should have `min="-50" max="50"` so that the middle is 0.

Reply:
I get it, seems I did not look good enough into the template :). Fixed with next version.